imports/pages/groups/finder/Result.js

```diff
@@ -205,6 +210,36 @@ const GROUP_FINDER_QUERY = gql`
   }
 `;
 
+const getDay = (schedule: String) => {
+  let day;
+  switch (schedule) {
+    case "sunday":
+      day = 0;
```

Comment:
I tried this I think when I first did the function, but I was getting an error (not sure what it was anymore) but changed it to the other way in favor of just getting it working. I updated it with returns and it is MUCH nicer.